ftp/src/main/scala/akka/stream/alpakka/ftp/javadsl/FtpApi.scala

```diff
@@ -149,7 +151,22 @@ sealed trait FtpApi[FtpClient] { _: FtpSourceFactory[FtpClient] =>
       chunkSize: Int = DefaultChunkSize
   ): Source[ByteString, CompletionStage[IOResult]] = {
     import scala.compat.java8.FutureConverters._
-    ScalaSource.fromGraph(createIOGraph(path, connectionSettings, chunkSize)).mapMaterializedValue(_.toJava).asJava
+    ScalaSource.fromGraph(createIOSource(path, connectionSettings, chunkSize)).mapMaterializedValue(_.toJava).asJava
+  }
+
+  /**
+   * Java API: creates a [[Sink]] of [[ByteString]] to some file [[Path]].
+   *
+   * @param path the file path
+   * @param connectionSettings connection settings
+   * @return A [[Sink]] of [[ByteString]] that materializes to a [[CompletionStage]] of [[IOResult]]
+   */
+  def toPath(
+      path: Path,
```

Comment:
I think that makes sense. String path is also used in the `ls` Sinks already. 

@juanjovazquez any particular reason why this was applied to `fromPath` ?

I can go ahead and  use String in both `fromPath` and `toPath`.